pkg/systemtemplate/template.go

```diff
@@ -337,6 +339,8 @@ spec:
           type: DirectoryOrCreate
   updateStrategy:
     type: RollingUpdate
+    rollingUpdate:
+	  maxUnavailable: 25%
```

Comment:
This has a tab before `maxUnavailable`

Reply:
The tab was even before the column where updateStrategy starts and doesn't show up in the IDE :/